app/views/purchases/_purchase_form.html.erb

```diff
@@ -14,12 +14,20 @@
 
     <div class="row">
       <div class="col-xs-8 col-md-6 col-lg-3">
-        <%= f.association :storage_location,
-                          collection: @storage_locations,
-                          label: "Storage Location",
-                          error: "Where is it being stored?",
-                          selected: current_organization.intake_location,
-                          wrapper: :input_group %>
+        <% if @purchase.new_record? %>
+          <%= f.association :storage_location,
+                            collection: @storage_locations,
+                            label: "Storage Location",
+                            error: "Where is it being stored?",
+                            selected: current_organization.intake_location,
```

Comment:
We can probably simplify this quite a bit with a ternary on the selected.
I imagine that something like:
`selected: @purchase.new_record? ? current_organization.intake_location : @purchase.storage_location,`
would work.
I bet you could even just do something like:
`selected: @purchase.new_record? ? current_organization.intake_location : nil,`

Reply:
Hi @seanmarcia, yes you are correct. Thank you for the suggestion, I've implemented the change that you suggested and is working fine.

Reply:
Thanks! I think the last part of the form to this would be to just move it into a helper :)

Reply:
No problem, created a helper function for it now.